web-services/query/src/main/java/datawave/webservice/query/logic/composite/CompositeQueryLogic.java

```diff
@@ -361,4 +403,82 @@ public Set<String> getExampleQueries() {
         }
         return params.isEmpty() ? null : params;
     }
+    
+    /**
+     * We can run the query if we can and at least of one of the children can.
+     * 
+     * @return canRunQuery
+     */
+    @Override
+    public boolean canRunQuery() {
+        boolean canRun = false;
+        if (super.canRunQuery()) {
+            for (QueryLogic<?> logic : this.queryLogics) {
+                canRun = logic.canRunQuery();
+                if (canRun) {
+                    break;
+                }
+            }
+        }
+        return canRun;
```

Comment:
Consider this slight simplification:
```suggestion
        if (super.canRunQuery()) {
            for (QueryLogic<?> logic : this.queryLogics) {
                if (logic.canRunQuery()) {
                    return true;
                }
            }
        }
        return false;
```

Reply:
sure.